cmd/snap/cmd_quota.go

```diff
@@ -411,8 +413,10 @@ func (x *cmdQuota) Execute(args []string) (err error) {
 			val := strings.TrimSpace(fmtSize(int64(group.Constraints.Journal.Size)))
 			fmt.Fprintf(w, "  journal-size:\t%s\n", val)
 		}
-		if group.Constraints.Journal.RateCount != 0 && group.Constraints.Journal.RatePeriod != 0 {
-			fmt.Fprintf(w, "  journal-rate:\t%d/%s\n", group.Constraints.Journal.RateCount, group.Constraints.Journal.RatePeriod)
+		if group.Constraints.Journal.QuotaJournalRate != nil {
```

Comment:
do we have a test for this?

Reply:
Yes, there is a test for a journal quota with rate limit set that hits this specific line